tests/unit/moduleapi/datatype.tcl

```diff
@@ -189,6 +189,20 @@ start_server {tags {"modules"}} {
                 } else {
                     fail "Unable to reduce the defragmentation speed."
                 }
+
+                # Create some small traffic to verify it does not restore defragmentation speed
+                for {set i 0} {$i < 100} {incr i} { 
+                    r set k dummy
+                    r del k
+                }
+                assert_equal [s active_defrag_running] 5
```

Comment:
don't we need to wait for cron to even consider changing?
maybe keep running the traffic for a minimum amount of time?
and also, the current workload, setting and deleting a key will probably have zero impact, maybe do some more fuzzing, or overwrites existing data?